package.json

```diff
@@ -107,7 +109,8 @@
     "rollup": "^4.6.0",
     "rollup-plugin-filesize": "^10.0.0",
     "rollup-plugin-visualizer": "^5.9.0",
-    "servez": "^2.0.0"
+    "express": "^4.17.2",
```

Comment:
not sure if we need `servez` anymore since we can test now in browser environment so I removed it.
Added `express` explicitly since it was used already in puppeteer script but not declared (ESlint warned about that). The lib was installed indirectly by some other dependency.
Added `cross-env` to support setting ENV vars in jobs cross platform.